### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by osnas**
Updated ## Installing the reference implementation -> ### Create a new image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,14 +279,18 @@ network={
 	psk="The password of your WiFi network"
 }
 ```
-* Move the microSD card from your workstation to the Raspberry-Pi and start the device
+The RaspberryPi will get an IP from the DHCP of your network.
 
-Then execute the following steps after obtaining an SSH connection
-to your Raspberry-Pi.
+* To have the changes written corectly, before the umount of the directories "rootfs" and "boot" type "sync" in a command line.
```
Comment (stefanos1316) [inline README.md]: Please change "corectly" to "correctly"

**Review by stefanos1316 — CHANGES_REQUESTED** [review]
Thank you for the commit! Please change the typo.

File: README.md (hunk 1 above)
Comment (dspinellis) [reply README.md]: `sync` is a Unix-dependent step (and it should be `unmount`).  Other OS's have different methods, so write a generic description.

File: README.md [hunk 2]
```diff
@@ -278,14 +279,18 @@ network={
 	psk="The password of your WiFi network"
 }
 ```
-* Move the microSD card from your workstation to the Raspberry-Pi and start the device
```
Comment (dspinellis) [inline README.md:281]: Isn't this line needed?

**Review by dspinellis — CHANGES_REQUESTED** [review]
Please address the few comments, and push a fresh commit encompassing them.

**Pull request closed without merging**